package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "geek-lab",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [
+    "cli"
+  ],
+  "author": "Rafael Prado",
+  "contributors": [
+    {
+      "name": "Francisco Silva",
+      "email": "silva.fass.developer@gmail.com"
+    }
+  ],
+  "license": "MIT",
+  "main": "./bin/geek-lab.js",
+  "scripts": {
+    "postinstall": "node ./src/scripts/post-install.js",
+    "lint": "eslint .",
+    "test": "nyc --reporter=html --reporter=text mocha --timeout 3000"
+  },
+  "pre-push": [
+    "lint",
+    "test"
+  ],
+  "bin": {
```

Comment:
any other ideas for bin shortcuts?

Reply:
![reason to not use gl alias](https://user-images.githubusercontent.com/11183347/71257242-f6305400-2311-11ea-95ee-0ce640da7c13.JPG)

Reply:
reason to not use gl :/